packages/xstate-svelte/test/package.json

```diff
@@ -1,3 +1,6 @@
 {
-  "type": "module"
+  "type": "module",
+  "devDependencies": {
+    "@xstate/svelte": "workspace:^"
+  }
```

Comment:
This seems fine? Not really sure

Reply:
Interesting, this is weird and it doesn't make sense (to me :P). This file doesn't define dependencies at all. On principle, I'd like to have this removed from here... but also it's probably not worth putting the effort into fixing this 🤷‍♂️